decksite/templates/editrules.mustache

```diff
@@ -0,0 +1,88 @@
+<h1>Deck Classification Rules</h1>
+<section>
+    <p>{{num_classified}} out of {{num_total}} decks currently covered by a rule.</p>
+
+    {{#has_doubled_decks}}
```

Comment:
nit. Spacing in templates should follow programmatic rules over making the HTML look perfect in "view source". So as this is functionally an if statement the contents should be indented four spaces.